download.py

```diff
@@ -2,17 +2,21 @@
 import heapq
 import hashlib
 import random
-
+from utils import pretty_print
+import time
 BLOCK_LENGTH = 2**14
 
 
 class DownloadHandler:
-    def __init__(self, tracker):
+    def __init__(self, tracker, torrent):
```

Comment:
needed to add the parent Torrent class so I could use it for computing the download avg speed for lines 48-61 and line 67-69